src/redux/books/bookSlice.js

```diff
@@ -1,23 +1,107 @@
-import { createSlice } from '@reduxjs/toolkit';
-import BookItems from './BookItems';
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// import BookItems from './BookItems';
+import axios from 'axios';
+
+const apiUrl =
+  'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/WluyIk7BUwsnpiFriH9k/books/';
+
 const initialState = {
-  bookItems: BookItems,
+  bookItems: [],
+  isLoading: false,
 };
 
+export const getBooks = createAsyncThunk('books/getBooks', async (thunkAPI) => {
+  try {
+    // console.log(thunkAPI);
+    const resp = await axios.get(apiUrl);
+    // to fetch data from API with the desired format
```

Comment:
- [ ] Please delete all commented-out code, it will make your codebase look unorganised and harder to read.